### PR TITLE
reportxml: added reportxml package

### PR DESCRIPTION
**Pull request by kononovn**

### CODE REVIEW

File: pkg/reportxml/reportxml.go [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+		_ = file.Close()
+	}()
+
+	encoder := xml.NewEncoder(file)
+	encoder.Indent("  ", "    ")
+
+	err = encoder.Encode(testCases)
+
+	if err != nil {
+		panic("failed to dump report to file")
```
Comment (achuzhoy) [inline pkg/reportxml/reportxml.go:238]: on line 225 the  panic param is with fmt.Errorf and here without.

File: pkg/reportxml/reportxml.go [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/ginkgo/v2/types"
+)
+
+type (
+	// TestSuite represents formatted test suite.
+	TestSuite struct {
+		XMLName    xml.Name   `xml:"testsuite"`
+		Name       string     `xml:"name,attr"`
+		Tests      int        `xml:"tests,attr"`
```
Comment (achuzhoy) [inline pkg/reportxml/reportxml.go:21]: TestsCount ?

Comment (kononovn) [reply pkg/reportxml/reportxml.go:21]: This is standard Junit report format. I can not change it

**Review by achuzhoy — APPROVED** [review]

**Review by cdvultur — APPROVED** [review]
/lgtm

**Pull request merged**